portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/structure/WebContentStructures.testcase

```diff
@@ -599,6 +599,16 @@ definition {
 			structureName = "Basic Web Content");
 	}
 
+	@description = "This is a test for LPS-125933. Create, read, update and delete a Structure with all possible fields."
+	@ignore = "true"
+	@priority = "5"
+	test ViewAndEditAsWellAsDeleteStructureWithTitleAndAllFields {
+		property portal.acceptance = "false";
+
+		// TODO LPS-123456 ViewAndEditAsWellAsDeleteStructureWithTitleAndAllFields pending implementation
```

Comment:
We need to change this LPS for the one related to the story we are automating. LPS-125933 in this case.

Same for all // TODO comments

Reply:
I see this is my fault since in my PR it is with TODO LPS-123456.. sorry :(

Reply:
Never mind. I already fix it locally :)